### ISSUE TITLE
Feature Request:  Corp overview > skill report (customizable)

### ISSUE THREAD

**Issue by sergiorochalopes** [eveseat/seat#198]
Request an customizable report overview, corp wise, on member count on skill name "X" with skill level "Y" and so on.

Thank you!

o/

**Comment by warlof** [comment]
Hello there,

what's customizable means for you ?
That perhaps the more important things in this request, since building a chart which is counting people per skill and associated level is really not the hardest :p

**Comment by justindh** [comment]
If I were going to use such a tool, I'd like to be able to specify a set of skills, say all scanning skills to atleast 4, and have a simple and clean view, of which characters have those skill sets. Bonus of it includes People groups.

**Comment by sergiorochalopes** [comment]
Yes, exactly. Also, for example, I would be interested in knowing... how many pilots can fly.. lets say a sabre.. or see/know how many pilots can fly a fit that i may upload.. in a report format of some sort, or a list, etc.. 
Thank you!

**Comment by justindh** [comment]
I might lean more towards a skillplan comparison, I've found in working on creating a Doctrine Manager, and even dealing with alternatives like Fleet-up, that while people can technically sit in a ship, they lack many fundamental skills. Knowing that some one has Interdictors 1, but not checking if they have shield comp, and propulsion support skills is bad. But trying to code for "support skills" is hard. I would limit the scope to allowing skill sets to be built, or imported via text or xml outputs of programs like Evemon, rather then parsing fits them selves to build the skill list.

**Comment by Crypta-Eve** [comment]
There is a couple of things that address this at the moment. You can use Squads in order to create filters on skill levels. Or to see who can fly ships / fits have a look at the seat-fittings plugin. Scope beyond that may be better in a new plugin.
